dashboard/v1.1/package.json

```diff
@@ -9,10 +9,8 @@
     "apexcharts": "^3.19.0",
     "material-table": "1.68.0",
     "material-ui-search-bar": "^1.0.0",
-    "moment": "^2.29.1",
```

Comment:
Either way `moment` is deprecated. We should use other alternatives like `day.js`.

Refer: https://momentjs.com/docs/#/-project-status/

Reply:
@chetanyakan react-datetime has a dependency on momentjs